CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [Unreleased]
 
+
+## [1.10.1] - 2016-07-02
```

Comment:
The peerDependency list is part of the public API exposed by any package, since it requires explicit developer interaction (manually installing + updating the peer dependency).